src/fs/types.ts

```diff
@@ -21,32 +23,32 @@ export interface Properties {
 }
 
 export interface Posix {
-  exists(path: DistinctivePath): Promise<boolean>
-  get(path: DistinctivePath): Promise<PuttableUnixTree | File | null>
-  mv(from: DistinctivePath, to: DistinctivePath): Promise<this>
-  rm(path: DistinctivePath): Promise<this>
+  exists(path: DistinctivePath<PartitionedNonEmpty<Partition>>): Promise<boolean>
```

Comment:
Are the `public` and `private` partitions guaranteed to exist? Is there any time that a developer would need to check for them? If so, this type may prevent them from doing so.